frontend/src/metabase/query_builder/components/AggregationPopover/AggregationPopover.jsx

```diff
@@ -143,11 +143,12 @@ export default class AggregationPopover extends Component {
   };
 
   _getAvailableAggregations() {
+    // XXX: This looks like the place we generate the options for the aggregation popover
     const { aggregationOperators, query, dimension, showRawData } = this.props;
     return (
       aggregationOperators ||
       dimension?.aggregationOperators() ||
-      query.table().aggregationOperators()
+      query.aggregationOperators()
```

Comment:
Even though `aggregationOperators` already exists in `StructuredQuery` before this PR, we still call the one in `Table` instead. Previously it'd be equivalent whether or not we call `query.aggregationOperators()` or `query.table().aggregationOperators()`, but not after this PR, since the new `StructuredQuery`'s `aggregationOperators` takes expression into account not only table fields when calculating which aggregation operators it would show in the aggregation step.

<img width="384" alt="image" src="https://user-images.githubusercontent.com/1937582/225985893-435026bd-eff5-4623-8818-685c9fa99db5.png">